cmd/oras/internal/display/content/manifest_fetch.go

```diff
@@ -31,14 +31,19 @@ type manifestFetch struct {
 	outputPath string
 }
 
-func (h *manifestFetch) OnContentFetched(desc ocispec.Descriptor, manifest []byte) error {
+func (h *manifestFetch) OnContentFetched(desc ocispec.Descriptor, manifest []byte) (err error) {
 	out := h.stdout
 	if h.outputPath != "-" && h.outputPath != "" {
 		f, err := os.Create(h.outputPath)
```

Comment:
It seems the `err` here will override the `err` at line 34.

Reply:
Renamed the variables.